com.unity.render-pipelines.universal/Runtime/2D/LightUtility.cs

```diff
@@ -1,11 +1,22 @@
 using System.Collections.Generic;
 using System.Linq;
+using Unity.Collections;
+using Unity.Mathematics;
 using UnityEngine.Experimental.Rendering.Universal.LibTessDotNet;
+using UnityEngine.Rendering;
+using UnityEngine.U2D;
 
 namespace UnityEngine.Experimental.Rendering.Universal
 {
     internal static class LightUtility
     {
+        // public static bool CheckForChange<T>(T a, ref T b) where T : struct
+        // {
+        //     var changed = !EqualityComparer<T>.Default.Equals(a, b);
+        //     b = a;
+        //     return changed;
+        // }
```

Comment:
I think we can leave only this generic version and get rid of all other overloads.

Reply:
i'm not sure I understand the performance characteristics of EqualityComparer